### DISCUSSION TITLE
I get errors when compiling using ESP-IDF

### DISCUSSION THREAD

**Discussion by ryan714465** [darthcloud/BlueRetro#110] [Q&A] [answered]
I tried to compile your project in the BlueRetro-1.1.1 version, but I get an error before finishing compiling 

[1098/1100] Linking CXX executable BlueRetro.elf
FAILED: BlueRetro.elf.

I have installed and uninstalled several versions of ESP-IDF and they all give me errors when compiling. ESP-UDF version 4.4. is the one that I get a single error

note: i am using windows 10

Which program and which version of it do you use to compile?

I need help

[95/97] Linking C executable bootloader.elf
[96/97] Generating binary image from built executable
esptool.py v3.2-dev
Merged 1 ELF section
Generated C:/Users/pc1/Documents/EspIDF/BlueRetro-1.1.1/build/bootloader/bootloader.bin
[97/97] cmd.exe /C "cd /D C:\Users\pc1\Documents\EspIDF\BlueRetro-1.1.1\build\bootloader\esp-idf\esptool_py && C:\.espressif\python_env\idf4.4_py3.8_env\Scripts\python.exe C:/esp-idf/components/partition_table/check_sizes.py --offset 0x8000 bootloader 0x1000 C:/Users/pc1/Documents/EspIDF/BlueRetro-1.1.1/build/bootloader/bootloader.bin"
Bootloader binary size 0x5a30 bytes. 0x15d0 bytes (19%) free.
[1098/1100] Linking CXX executable BlueRetro.elf
FAILED: BlueRetro.elf
cmd.exe /C "cd . && C:\.espressif\tools\xtensa-esp32-elf\esp-2021r2-8.4.0\xtensa-esp32-elf\bin\xtensa-esp32-elf-g++.exe -mlongcalls -Wno-frame-address  @CMakeFiles\BlueRetro.elf.rsp -o BlueRetro.elf  && cd ."
c:/.espressif/tools/xtensa-esp32-elf/esp-2021r2-8.4.0/xtensa-esp32-elf/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld.exe: esp-idf/main/libmain.a(core0_stall_highint_hdl.S.obj): in function `xt_highint5':
c:\users\pc1\documents\espidf\blueretro-1.1.1\build/../main/system/core0_stall_highint_hdl.S:48: multiple definition of `xt_highint5'; esp-idf/esp_system/libesp_system.a(highint_hdl.S.obj):C:/esp-idf/components/esp_system/port/soc/esp32/highint_hdl.S:221: first defined here
collect2.exe: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
ninja failed with exit code 1
PS C:\Users\pc1\Documents\EspIDF\BlueRetro-1.1.1>

**Comment by darthcloud** [comment, marked as answer]
Read that first https://github.com/darthcloud/BlueRetroRoot 
(btw link to that is on the README.md under Project documentation section -> Development environment setup & building instruction)

Instruction are for Linux so you will likely need to adapt a bit most likely.
